AirCasting/AppState.swift

```diff
@@ -0,0 +1,26 @@
+// Created by Lunar on 17/08/2021.
+//
+import SwiftUI
+
+class AppState: ObservableObject {
```

Comment:
it's only for the microphone sessions while mobile sessions can be both mic or AB :)

Reply:
Yes You are totally right!